cli/lsp/analysis.rs

```diff
@@ -479,6 +479,91 @@ impl CodeActionCollection {
     Ok(())
   }
 
+  pub(crate) fn add_deno_lint_ignore_action(
+    &mut self,
+    specifier: &ModuleSpecifier,
+    diagnostic: &lsp::Diagnostic,
+  ) -> Result<(), AnyError> {
+    let code = diagnostic
+      .code
+      .as_ref()
+      .map(|v| match v {
+        lsp::NumberOrString::String(v) => v.to_owned(),
+        _ => "".to_string(),
+      })
+      .unwrap();
+
+    let mut changes = HashMap::new();
+    HashMap::insert(
+      &mut changes,
+      specifier.clone(),
+      vec![lsp::TextEdit {
+        new_text: format!("// deno-lint-ignore {}\n", code),
+        range: lsp::Range {
+          start: lsp::Position {
+            line: diagnostic.range.start.line,
+            character: 0,
```

Comment:
There is a way to get the indentation at the current position via tsserver ("indentaiton" command), but I don't think that would be helpful here though. We may just want to get the index of the first non-whitespace character on the current line and that will be good enough for 99% of cases.

Reply:
Ah, this makes sense. I'll see how I can get the index of the first non-whitespace character of the start line.

Thanks for the suggestion! :)